UDPClientGUI/SiceUDPClient.cs

```diff
@@ -78,5 +79,11 @@ public void Handle(ClientCommandEvent Command)
             handler[Command.ClientCommand]();
         }
 
+        public void Handle(ClientLoginEvent message)
+        {
+            if (!ConnectionStatus) return;
+            var loginMessage = Encoding.ASCII.GetBytes(JsonConvert.SerializeObject(message));
+            client.Send(loginMessage, loginMessage.Length);
```

Comment:
Isn't this an async method? If this takes a while (can be up to 1 second), prefer to have an async method instead. And then disable the buttons to make sure user doesn't click it twice.